### PR TITLE
basket ok

### PR DESCRIPTION
**Pull request by bayram-69**

### CODE REVIEW

**Review by Jhanemba64 — APPROVED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-  const [basketCount, setBasketCount] = useState(0);
   return (
     <div className="App">
-      <Navbar basketCount={basketCount} setBasketCount={setBasketCount} />
-      <Outlet />
-      <Footer />
+      <BasketContextProvider>
+        <Navbar />
+        <Outlet />
+        <Footer />
```
Comment (PierreA44) [inline frontend/src/App.jsx:14]: The Footer doesn't need to be in the BasketContextProvider!

**Review by PierreA44 — COMMENTED** [review]
check app.jsx

**Review by PierreA44 — APPROVED** [review]

File: frontend/src/components/shopList/ShopList.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        {cardItems.map((item) => (
+          <div key={item.id} className={styles.cartItems}>
+            <div className={styles.cartItems_img}>
+              <img src={item.image} alt={item.nameItem} />
+              <p>{item.nameItem}</p>
+            </div>
+            <div className={styles.cartItems_info}>
+              <div className={styles.buttonsItems}>
+                <button type="button"> + </button>
+                <span> &nbsp;&nbsp;{item.quantity}&nbsp;&nbsp;</span>
```
Comment (ioayoub) [inline frontend/src/components/shopList/ShopList.jsx:23]: ???
use css here instead of crazy &nbsp;

**Review by ioayoub — CHANGES_REQUESTED** [review]
Conflicts to fix + &nbsp; in ShopList.jsx

**Review by akornne — APPROVED** [review]

**Review by ioayoub — CHANGES_REQUESTED** [review]
Conflicts to fix

**Review by ioayoub — APPROVED** [review]

**Pull request merged**